api/src/membership/models.py

```diff
@@ -202,20 +218,20 @@ def __repr__(self) -> str:
 class PhoneNumberChangeRequest(Base):
     __tablename__ = "change_phone_number_requests"
 
-    id = Column(Integer, primary_key=True, nullable=False, autoincrement=True)
-    member_id = Column(Integer, ForeignKey("membership_members.member_id"), nullable=True)
-    phone = Column(String(255), nullable=False)
```

Comment:
The string size constraint was removed in a few places, without a migration.

Reply:
The string size is only used if a table needs to be created by sqlalchemy. But since we never do that (it's always created by our migration sql files), the length is not necessary, I believe.

Reply:
I think it makes sense to keep the full description in the code so that it represents the intended current state of the database. It's difficult to decipher that from the migration scripts, and looking in the database itself is also painful.

Another benefit of that, is that it then becomes possible to create the database from the declarative ORM in Python using `Base.metadata.create_all(engine)`. While it also makes it easier to switch to a tool like Alembic, which creates types/constraints for the database migrations based on the Python code itself (which I think would be an improvement from the current migration scripts).

Reply:
Fair enough. I don't think all our annotations are accurate. But I suppose it's better to not make it worse.